### DISCUSSION TITLE
Diving Suits Pickup Button

### DISCUSSION THREAD

**Discussion by OmegaSybers** [FakeFishGames/Barotrauma#6110] [Ideas and suggestions]
Currently it is sometimes very difficult to pickup diving suits and toolbelts without equipping them. Maybe adding a dedicated pickup button like E or G would help

**Comment by MystLeissa** [comment]
Probably could be easily fixed by reversing the order in which it goes to your outfit slot when dbl clicking. Right now it first goes to your body, then your hands, if it went to your hands first, it'd not only be more realistic (e.g. "Dawning the Suit") it'd also fix the problem where an empty oxygen supply kills you when trying to carry an unequipped suit.

**Comment by Rokvach** [comment]
Diving suits can be pick-up with E right now, you just can't do it whilst the suit is inside a locker. So you need to drop it to the ground and then press E to pick it up.
